src/config/cuba/index.ts

```diff
@@ -9,5 +9,6 @@ export default {
   rawLayers,
   rawTables,
   translation,
+  defaultUrl: 'prism-cuba.org',
```

Comment:
This URL does not exist, so why put it in here?

Reply:
for tests, sorry, forgot to delete it(